src/components/subtitlesettings/subtitleappearancehelper.js

```diff
@@ -103,10 +103,10 @@ function getTextStyles(settings, preview) {
         const lineHeight = 1.35; // FIXME: It is better to read this value from element
         const line = Math.abs(pos * lineHeight);
         if (pos < 0) {
-            list.push({ name: 'min-height', value: `${line}em` });
+            list.push({ name: 'margin-bottom', value: `${line - 1}em` });
```

Comment:
Shouldn't `lineHeight` be subtracted instead of `1` to zero out the bottom margin?